github4s/src/main/scala/github4s/algebras/Repositories.scala

```diff
@@ -271,6 +271,22 @@ trait Repositories[F[_]] {
       headers: Map[String, String] = Map()
   ): F[GHResponse[List[User]]]
 
+  /**
+   * Get a single release
+   *
+   * @param releaseId id of the release
+   * @param owner of the repo
+   * @param repo name of the repo
+   * @param headers optional user headers to include in the request
+   * @return a GHResponse with List[Release]
+   */
+  def getASingleRelease(
```

Comment:
I think we can rename that to simply `getRelease`

Reply:
You are absolutely right, I just used the name which is given in github api documentation
https://developer.github.com/v3/repos/releases/#get-a-single-release